src/lib/y2users/inst_users_dialog_helper.rb

```diff
@@ -0,0 +1,129 @@
+# Copyright (c) [2021] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+require "y2users"
+require "y2users/users_simple"
+
+module Y2Users
+  # Mixin for user and password validations based on Y2Users for isntallation dialogs
+  module InstUsersDialogHelper
+    Yast.import "UI"
+    Yast.import "Popup"
+    Yast.import "Report"
+
+    # Needed to be able to call textdomain below
+    extend Yast::I18n
+
+    def self.included(_mod)
+      # Needed to prevent the automatic checks (rake check:pot) from complaining
```

Comment:
well, it is not just about automatic bots. It has real reason. gettext we use to extract localized messages for translators use textdomain call in given file to associate textdomain with text. That allows us to have more textdomain in single repo ( like Yast2 ) So if it is not there, pot file will miss messages from this file.

Reply:
Oh, thanks for the explanation. I'm going to update the comment, which I stolen from https://github.com/yast/yast-users/blob/54a66f550e9ccf0d002e21410238a222166a096d/src/lib/y2users/help_texts.rb#L31-L37